docs/transparent-validation/00_templates/laboratory-stand-assembly-guide-template.md

```diff
@@ -0,0 +1,124 @@
+# Laboratory stand dedicated to MSI PRO Z690-A assembly guide
+
+## Introduction
+
+This document describes the assembly procedure dedicated to [DEVICE_NAME]
+testing stand.
+
+## Prerequisites
+
+The below table contains information about all elements which are needed to
+create the testing stand.
+
+* [device]
+* [element1]
+* [element2]
+
+## Pre-setup activities
+
+The following subsections describe the method of preparing all the
+components of the laboratory stand.
+
+### [DEVICE_NAME]
+
+[DEVICE_NAME] platform should be prepared in accordance with the
+[Assembly](some.md#some-section) documentation.
+
+### [EG.] RTE
+
+RTE (acronym: Remote Testing Environment) should be prepared in accordance with
+[Quick start guide](../rte/v1.1.0/quick-start-guide.md) documentation dedicated
+to the device.
+
+### [EG.] Sonoff
+
+Sonoff should be prepared in accordance with
+[Quick start guide](../sonoff/quick-start-guide.md) documentation dedicated to
+the device.
+
+## Connections
+
+The following sections describe how to enable all of the following features:
+
+* serial connection to the platform,
+* controlling power supply,
+* enabling basic power actions with the platform (power off/power on/reset),
+* external flashing with the RTE.
+* [optional-connections]
+
+### Serial connection
+
+1. ...
+1. ...
+
+### Power supply controlling
+
+1. ...
+1. ...
+
+### Basic power operations enabling
+
+1. ...
+1. ...
+
+### External flashing enabling
+
+1. ...
+1. ...
+
+### Complete Setup
+
+[EXAMPLE:]
+
+After preparing all of the connections also three activities should be
+performed to enable all of the test stand features:
+
+1. Connect Sonoff to the mains:
+
+    ![IMG](images/sonoff_connected.jpg)
+
+1. Connect the RTE to the Internet by using the Ethernet cable.
+1. Connect the RTE to the mains by using the microUSB 5 V/2 A power supply.
+1. [optional-step]
+
+Complete setup should looks as follows:
+
+![Complete](images/some-image.jpg)
```

Comment:
If the file `some-image` indeed exists, it should be renamed.

Reply:
Updated